fusion-builder/fusion-builder-fr_FR.po

```diff
@@ -126,7 +129,7 @@ msgstr ""
 
 #: inc/admin-screens/faq.php:14
 msgid "What Is The Fusion Builder?"
-msgstr ""
+msgstr "Qu'est-ce que le Fusion Builder ?"
```

Comment:
Should this not be Qu'est-ce que c'est?

Reply:
No this is correct :) if you want to say "qu'est-ce que c'est?" then you simply ask "what is it?"

Reply:
or if you want to be more Frenchy, remove the "le" --> "Qu'est-ce que Fusion Builder ?", for me this is more correct.

Reply:
Perhaps that's what wasn't sitting right with me... Removing the "le" and merging. Thank you for educating me :+1: